v1.0/brainAtlasVersion.schema.json

```diff
@@ -14,68 +14,71 @@
   "properties": {
     "alias": {
       "type": "string",
-      "description": "User-defined identifier, often a short name, of this brain atlas version."
+      "_instruction": "Enter the user-defined identifier for this brain atlas version (e.g. a short name)."
     },
     "alternativeVersionOf": {
       "type": "array",
-      "description": "Link to an alternative brain atlas versions.",
+      "_instruction": "Add one or several alternative versions of this brain atlas versions.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/sands/BrainAtlasVersion"
-      ]
+      ],
+      "minItems": 1,
+      "uniqueItems": true
     },
     "annotationSet": {
       "type": "array",
-      "description": "List of annotations that define this brain atlas version",
+      "_instruction": "Add one or several annotations included in this brain atlas version.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/sands/Annotation"
-      ]
+      ],
+      "minItems": 1,
+      "uniqueItems": true
     },
     "coordinateSpace": {
-      "type": "object",
-      "description": "Link to coordinate space this brain atlas version is represented in",
+      "_instruction": "Add the coordinate space in which this brain atlas version exists in.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/sands/CoordinateSpace"
       ]
     },
     "fullName": {
       "type": "string",
-      "description": "A descriptive name of this brain atlas."
+      "_instruction": "Enter a descriptive name for this brain atlas version."
     },
     "newVersionOf": {
-      "description": "Link to the previous brain atlas version.",
+      "_instruction": "Add the earlier version this brain atlas version is a new version of.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/sands/BrainAtlasVersion"
       ]
     },
     "referenceDataset": {
```

Comment:
Good question...I think I wasn't in the meeting when this was decided to be added. I was never 100% sure what was supposed to be added here. I wouldn't oppose allowing several. This would make it more flexible. Then everything that was used to create the brain atlas version could be stored as one or several datasets. If that's something that should never happen anyway, we could keep it as count = 1. @aeidi89 DO you have any opinons about this?